sidekiq_autoscale.gemspec

```
@@ -18,15 +18,16 @@ Gem::Specification.new do |s|
 
   s.files = Dir["{app,config,db,lib}/**/*", "MIT-LICENSE", "Rakefile", "README.md"]
 
+  s.add_dependency "addressable", ">= 2.8.0"
```

```suggestion
  s.add_dependency "addressable", "~> 2.8"
```

Using `~>` may be better, in the event that `addressable v3.0` introduces breaking errors.